src/system.ts

```diff
@@ -361,13 +361,31 @@ export const contiguousDataSource = new ReadThroughDataCache({
   dataContentAttributeImporter,
 });
 
+export const dataItemIndexer = new DataItemIndexer({
+  log,
+  eventEmitter,
+  indexWriter: dataItemIndexWriter,
+});
+
+const ans104DataIndexer = new Ans104DataIndexer({
+  log,
+  eventEmitter,
+  indexWriter: nestedDataIndexWriter,
+  contiguousDataIndex,
+});
+
+const shouldUnbundleDataItems = () =>
+  ans104DataIndexer.queueDepth() < config.MAX_DATA_ITEM_QUEUE_SIZE ||
```

Comment:
Let's make this `&&` so that we unbundle only when both are below the threshold.

Reply:
[`e82c796` (#190)](https://github.com/ar-io/ar-io-node/pull/190/commits/e82c796a875b57a83859a90fc47c8b6c3ddf10c3)